azext_edge/edge/providers/orchestration/instances.py

```diff
@@ -0,0 +1,75 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from typing import List, Optional
+
+from knack.log import get_logger
+from rich import print
+
+from ...util.az_client import get_iotops_mgmt_client, parse_resource_id
+from ...util.queryable import Queryable
+
+logger = get_logger(__name__)
+
+
+QUERIES = {
+    "get_cl_from_instance": """
+        resources
+        | where type =~ 'microsoft.extendedlocation/customlocations'
+        | where id =~ '{resource_id}'
+        | project id, name, properties
+        """
+}
+
+INSTANCES_API_VERSION = "2021-10-01-privatepreview"
+# TODO temporary
+BASE_URL = "https://eastus2euap.management.azure.com"
+QUALIFIED_RESOURCE_TYPE = "Private.IoTOperations/instances"
```

Comment:
This can be baked into the ResourceTypeMapping/ResourceProviderMapping enums

Reply:
This is a temporary mapping for this intermediate PR to work with private rp. In the future will evaluate if an enum makes sense, which a datapoint will be which other components other than the mgmt class needs to know about a typing. If other operations don't then there isn't a reason to have an enum.